packages/query/src/stacks/token-metadata/fungible-tokens/fungible-token-metadata.query.ts

```diff
@@ -22,20 +22,41 @@ const queryOptions = {
   staleTime,
 };
 
-function fetchFungibleTokenMetadata(client: StacksClient, signal: AbortSignal) {
-  return (principal: string) =>
-    client.getFtMetadata(principal, signal) as unknown as FtAssetResponse;
+export function fetchFungibleTokenMetadata(client: StacksClient, signal: AbortSignal) {
```

Comment:
With this pattern, should `signal` be optional, otherwise if we do call outside RQ, and we don't need cancellation, we'll have to pass an empty `new AbortController()`

Reply:
I don't see that we make the signal optional anywhere currently, is that something I should address at the client level if I remove the extra fetch function? Or, do you all want to keep that additional fetch function with each query?